dnf-behave-tests/features/module-list.feature

```diff
@@ -148,3 +148,11 @@ Scenario: I can limit the scope of disabled modules through providing specific m
         | dnf-ci-fedora-modular-updates | postgresql    | 10 [x]     | client, server, default     |
         | dnf-ci-fedora-modular-updates | postgresql    | 11 [x]     | client, server, default     |
 
+
+Scenario: Modules are ordered by repository then pakcage name and stream name
```

Comment:
The sorting key is more likely "module name" then package name.

Reply:
Yes, I have updated it, thank you.